### DISCUSSION TITLE
Should i upgrade to pro [GITHUB SUPPORT] ??

### DISCUSSION THREAD

**Discussion by asmawi117** [community/community#122037] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

My ticket has been open for 80 days, and I've yet to receive any response from the support team. Would upgrading to the Pro plan result in quicker assistance?

**Comment by Yackens** [comment]
Upgrading to a GitHub Pro plan might offer you additional features and benefits, but it doesn't guarantee quicker support response times. GitHub's support prioritization generally depends on the severity of the issue and the type of plan you have. While a Pro plan might provide access to higher-tier support channels or faster response times, this isn't explicitly guaranteed.

Steps to Consider:
- Follow Up on the Ticket: If you haven't done so already, try following up on the ticket. Sometimes, a polite reminder can bring attention to your issue.
- Community Assistance: Consider seeking help from the GitHub Community Forums or other developer communities. While this isn't a direct support channel, other users or GitHub staff might be able to assist.
